README.md

```diff
@@ -2,6 +2,40 @@
 
 Enables validation for inputs in conjunction with hyper-validator modules.
 
+## Usage
+
+This is the base for all the validator modules. It collects the different validator modules and runs the
+validattions at the right time. To install it simply do `npm install hyper-validator-base --save-dev`.
+This will add the validator base in you node_modules folder and insert it as a dependecy in `package.json`.
+
+Please remember to load the file in you view, its as simple as adding:
+
+```
+<script type="text/javascript" src="node_modules/hyper-validator-base/src/hyper-validator-base.js"></script>
+```
+
+On slightly bigger projects this might be automated using Gulp or Grunt.
+
+To initialize it you need to inject `'hyper-input-validator'` as a dependecy in your angular module. Like this:
```

Comment:
Should we go for either quotes or code formatting here? Both seem like overkill.